### PR TITLE
0.16.0 changelog

### PR DESCRIPTION
**Pull request by ddjjia**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -6,6 +6,29 @@ weight: 35
...
+- Management cluster upgrades will only move management cluster’s components to bootstrap cluster and back. ([#5914](https://github.com/aws/eks-anywhere/pull/5914))
+
+### Fixed
+- CloudStack control plane host port is only defaulted in CAPI objects if not provided. ([#5792](https://github.com/aws/eks-anywhere/pull/5792)) ([#5736](https://github.com/aws/eks-anywhere/pull/5736))
+
+### Deprecated
+- Add warning to deprecate disableCSI through CLI ([#5918](https://github.com/aws/eks-anywhere/pull/5918)). Refer to [documentation](https://anywhere.eks.amazonaws.com/docs/reference/clusterspec/vsphere/#disablecsi-optional-deprecated) for more information
+
+### Removed
+- Kuberenets 1.22 support
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/changelog.md:30]:
```suggestion
- Kubernetes 1.22 support
```

File: docs/content/en/docs/reference/changelog.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ weight: 35
 
 ## Unreleased
 
+## [v0.16.0](https://github.com/aws/eks-anywhere/releases/tag/v0.16.0)
+
+### Added
+- Workload clusters full lifecycle API support for CloudStack provider ([#2754](https://github.com/aws/eks-anywhere/issues/2754))
+- Enable proxy configuration for Bare Metal provider. ([#5925](https://github.com/aws/eks-anywhere/issues/5925))
```
Comment (jacobweinstock) [inline docs/content/en/docs/reference/changelog.md:13]: none of the other lines have a `.`, might want to remove this one.

File: docs/content/en/docs/reference/changelog.md [hunk 3]
```diff
@@ -6,6 +6,30 @@ weight: 35
...
+- Kubernetes 1.27 support ([#5929](https://github.com/aws/eks-anywhere/pull/5929))
+- Support for upgrades for clusters with pod disruption budgets ([5697](https://github.com/aws/eks-anywhere/pull/5697))
```
Comment (jacobweinstock) [inline docs/content/en/docs/reference/changelog.md:15]: the other items in the list have a `#` before the issue number. This one does not. might want to add it for consistency.

**Review by jacobweinstock — APPROVED** [review]

**Review by chrisnegus — COMMENTED** [review]
/lgtm

**Pull request merged**